README.md

```diff
@@ -1,20 +1,19 @@
-# Twilio React Video App
+# Twilio Video React App
 
-[![CircleCI](https://circleci.com/gh/twilio/twilio-video-app-react.svg?style=svg&circle-token=9d6b1e89d148181aaa6874c29849c730b8ca406d)](https://circleci.com/gh/twilio/twilio-video-app-react)
+[![CircleCI](https://circleci.com/gh/twilio/twilio-video-app-react.svg?style=shield&circle-token=9d6b1e89d148181aaa6874c29849c730b8ca406d)](https://circleci.com/gh/twilio/twilio-video-app-react)
 
 This application demonstrates a multi-party video application built with [twilio-video.js](https://github.com/twilio/twilio-video.js) and [Create React App](https://github.com/facebook/create-react-app).
 
 ![App Preview](https://user-images.githubusercontent.com/12685223/76361972-c035b700-62e5-11ea-8f9d-0bb24bd73fd4.png)
 
 ## Features
 
-- Video conferencing with real-time video and audio
-- Enable/disable camera
-- Mute/unmute microphone
-- Screen sharing
-- [Dominant speaker](https://www.twilio.com/docs/video/detecting-dominant-speaker) detection
-- [Network quality](https://www.twilio.com/docs/video/using-network-quality-api) indicator
-- [Bandwidth Profile API](https://www.twilio.com/docs/video/tutorials/using-bandwidth-profile-api)
+- [x] Video conferencing with real-time video and audio
+- [x] Enable/disable camera
+- [x] Mute/unmute mic
+- [x] Screen sharing
+- [x] [Dominant speaker](https://www.twilio.com/docs/video/detecting-dominant-speaker) indicator
+- [x] [Network quality](https://www.twilio.com/docs/video/using-network-quality-api) indicator
```

Comment:
I'm ok to add the boxes but let's keep Bandwidth Profile in there.

Reply:
Done